javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -140,6 +140,12 @@ private TreeUtils() {
      */
     private static final @Nullable Method TREEMAKER_SELECT;
 
+    /**
+     * The {@code JCTree.JCVariableDecl.declaredUsingVar()} method. We should use this method to
+     * determine if a variable is declared using {@code var} in Java 11+.
```

Comment:
```suggestion
     * The {@code JCTree.JCVariableDecl.declaredUsingVar()} method. This method
     * determines if a variable is declared using {@code var} in Java 17+.
```

Probably needs to be reformatted.